app/models/declaration/TransportInformationContainerData.scala

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.declaration
+
+import forms.MetadataPropertiesConvertable
+import forms.declaration.TransportInformationContainer
+import play.api.libs.json.Json
+
+case class TransportInformationContainerData(containers: Seq[TransportInformationContainer])
```

Comment:
So we're coming back to old solution with model for view and model for cache?

Reply:
not yet  I am leaving with what's built already and removing the ones that are being added are built new